### DISCUSSION TITLE
Drag & Drop for Record-Ordering

### DISCUSSION THREAD

**Discussion by jony1993** [avo-hq/avo#1062] [Feature requests]
Hi there,

The provided record ordering i(https://docs.avohq.io/2.0/resources.html#records-ordering) is already good. But it would be nice to have Drag&Drop to order records. This would make it even better and speed up the content editing a lot

**Comment by adrianthedev** [comment]
Yes, that's not a bad idea. 

The current re-ordering feature is on it's first iteration. It can definitely be improved. 

The drag an drop definitely adds a bit of JS complexity, but it's manageable. 

I'll add it to the roadmap. Maybe someone can lens a hand.

**Comment by Paul-Bob** [comment]
https://docs.avohq.io/3.0/records-reordering.html#reorder-using-drag-and-drop
